base_layer/core/src/transaction.rs

```diff
@@ -30,17 +30,13 @@ use crate::{
 };
 
 use crate::types::SignatureHash;
-use crypto::{
-    challenge::Challenge,
-    commitment::HomomorphicCommitment,
-    common::{Blake256, ByteArray},
-    hash::Hashable,
-};
+use crypto::{challenge::Challenge, commitment::HomomorphicCommitment, common::Blake256};
 use curve25519_dalek::scalar::Scalar;
 use derive::HashableOrdering;
 use derive_error::Error;
 use digest::Digest;
 use std::cmp::Ordering;
+use tari_utilities::{byte_array::ByteArray, hash::Hashable};
```

Comment:
How about exporting these traits on the root level? Should improve the ergonomics a bit. 
And maybe all loose functions (e.g. `to_hex`) can be grouped in a functions mod? 

Note: Keep the code organisation the same, just export them like this.